### PR TITLE
Feature/permit scraper

### PR DESCRIPTION
**Pull request by chrise96**

### CODE REVIEW

File: submit_to_sia.py [hunk 1]
```diff
@@ -151,6 +151,8 @@ def _image_upload(auth_headers: Dict[str, str], filename: str, sig_id: str) -> A
     conn, cur = upload_to_postgres.connect()
 
     # Get images with a detection
+    # TODO join with other table to check date?
+    # TODO filter max n items based on score
```
Comment (epureanudiana) [inline submit_to_sia.py:155]: this is work in progress, right?

File: visualizations/utils.py [hunk 2]
```diff
@@ -119,25 +85,35 @@ def is_container_permit(permit: Any) -> bool:
...
             and is_permit_valid_on_day(item)
         ):
             try:
-                # todo: use split_dutch_street_address(item.find("TEXT6").text)
-                address = remove_postal_code(item.find("TEXT6").text).replace(
-                    " ", "%20"
-                )
+                if len(item.getchildren()[-1]) > 0:  # Check if c_object exists
+                    address = item.getchildren()[-1].getchildren()[0]
+                    address_format = address.find("TEXT8").text + " " + address.find("INITIALS").text
```
Comment (epureanudiana) [inline visualizations/utils.py:94]: I think Evelien told you that in some(actually a lot) of the cases the address is not in TEXT8, but in other fields? Do you know about this?

File: submit_to_sia.py (hunk 1 above)
Comment (chrise96) [reply submit_to_sia.py:155]: Fixed in just merged branch, will remove and rebase now

File: visualizations/utils.py (hunk 2 above)
Comment (epureanudiana) [reply visualizations/utils.py:94]: Also, what is this line doing?

https://github.com/Computer-Vision-Team-Amsterdam/Detecting-Heavy-Objects/blob/b59c32fb10b540406b014fc8a3630315b124ea2c/visualizations/utils.py#L85

Comment (chrise96) [reply visualizations/utils.py:94]: Yes, good one. Because of that conversation, we implement it now this way

**Review by epureanudiana — COMMENTED** [review]
Left some comments!

**Pull request merged**